pkg/utils/has/controller.go

```diff
@@ -92,7 +92,13 @@ func (h *SuiteController) DeleteHasComponent(name string, namespace string) erro
 }
 
 // CreateComponent create an has component from a given name, namespace, application, devfile and a container image
-func (h *SuiteController) CreateComponent(applicationName, componentName, namespace, gitSourceURL, containerImageSource, outputContainerImage string) (*appservice.Component, error) {
+func (h *SuiteController) CreateComponent(applicationName, componentName, namespace, gitSourceURL, containerImageSource, outputContainerImage, secret string) (*appservice.Component, error) {
+	var containerImage string
+	if outputContainerImage != "" {
+		containerImage = containerImageSource
+	} else {
+		containerImage = containerImageSource
+	}
```

Comment:
I'm struggling to understand this condition 😅

Reply:
Typo 😆 . Though with your suggestion to just use a single parameter for the container image, this condition isn't really needed.